src/components/buttons/Button.jsx

```diff
@@ -0,0 +1,159 @@
+// @flow
```

Comment:
strict?

Reply:
I don't believe that adding `flow strict` is better than `flow` and it shouldn't be treated as request for changes in entire PR...

Reply:
You can write untyped or untested code for your own features, but common modules which will be used by everyone should be done in a proper way. I don't ask for `//@ flow strict` in PRs for daily tasks. But this thing will be used by everyone and should be typed in a good way.

Reply:
@PinkaminaDianePie I will change it to `//@ flow strict`, as you requested. I really don't remember when, or if ever, I merged code, without solving feedback or comments in the PR, so I guess we could avoid `request changes` flag. But no worries, if you prefer this way.
Also, please keep in mind, that the goal for all of us is to write high quality code, so I think there is no reason to suggest otherwise.
Anyways, I'm on my way with changes, I'll ping you when I'm done. Thx a lot for feedback.

Reply:
@PinkaminaDianePie done - changed to `//@ flow strict`, I have made some adjustments for `icon` and `href`,  pls have a look if you will have a moment. thx.

Reply:
@burakukula actually request changes was not because of `strict` but because of documentation - it's more important than `strict`. We introduced JSX with literals and without importing constants, we don't event reexport any constants from `index.js`. If we want to be consistent we shouldn't use the old way in the documentation. Otherwise, we will end up with both approaches in the same codebase forever. Please forgive me if `request changes` looked offending for you - I won't use it next time :) My intention was not to block PR from merging but to stress out that issues I mentioned are really important. But next time I will express it in words, not in some flags :)

Reply:
@PinkaminaDianePie 👍 no worries, all good.